examples/container_registry/BUILD

```diff
@@ -0,0 +1,14 @@
+container_registry(
+    name="exmaple-registry",
+    registry="example.com/docker/registry",
```

Comment:
This would more likely be tldn like `docker-registry.example.com:443` (it may be outdated now, but iirc, you have to include the port in custom registry url's, prob for disambiguation purposes)